### ISSUE TITLE
slither.io not working? 

### ISSUE THREAD

**Issue by jsastudent1** [wakestudents/IssueTracker#32]
i just get a error code whenever i try to load in is it just the game or computer?

**Comment by isteiger** [comment]
the proxy is down, thats why. it won't be coming back up so I need to remove that and a couple other games

**Comment by jsastudent1** [comment]
yikes, so why did the proxy go down?

**Comment by isteiger** [comment]
dad says so, i cant do anything about that, sorry

**Comment by jsastudent1** [comment]
dang did the school just find out and make you shut it down?

**Comment by isteiger** [comment]
no, my dad is worried about people doing illegal stuff on it and me being liable, which is a fair point

**Comment by jsastudent1** [comment]
fair enough i get it are basically all the html games gone?

**Comment by isteiger** [comment]
only the multiplayer ones really because i can't host those myself
